WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -114,17 +119,41 @@ public CoinViewModel(CoinListViewModel owner, SmartCoin model)
 
 			ToggleDetails = ReactiveCommand.Create(() => IsExpanded = !IsExpanded);
```

Comment:
This is not used. Remove it.

Reply:
@molnard done, please check latest commit